src/Components/SysDetailsTable/NewSysDetailsTable.js

```diff
@@ -253,16 +253,23 @@ const NewSysDetailsTable = ({ systemId }) => {
               ),
             },
             {
-              title: (
-                <span>
-                  {data.newMatchCount ? (
-                    <Icon style={{ marginRight: '8px' }}>
+              title:
+                data.newMatchCount > 0 ? (
+                  <span>
+                    <Icon
+                      data-testid="new-matches-bell-icon"
+                      status="info"
+                      style={{ marginRight: '8px' }}
+                    >
                       <BellIcon />
                     </Icon>
-                  ) : null}
-                  {data.newMatchCount?.toLocaleString()}
-                </span>
-              ),
+                    {data.newMatchCount?.toLocaleString()}
+                  </span>
+                ) : (
+                  <span style={{ color: '#d2d2d2' }}>
```

Comment:
This needs to be pf-v5-u-disabled-color-200. We shouldnt hard code in hex values.